eul/Views/Menu/StatusMenuView.swift

```diff
@@ -15,6 +15,18 @@ struct StatusMenuView: SizeChangeView {
 
     var onSizeChange: ((CGSize) -> Void)?
     var body: some View {
+        if preferenceStore.appearanceMode == .light {
+            base
+                .environment(\.colorScheme, .light)
+        } else if preferenceStore.appearanceMode == .dark {
+            base
+                .environment(\.colorScheme, .dark)
+        } else {
+            base
+        }
+    }
+
```

Comment:
something like 

```swift
if let appearanceMode = preferenceStore.appearanceMode {
    base.environment(...)
} else {
    base
}
```

Reply:
I think it's a good scenario for using `ViewModifier`

Reply:
Sorry but I don't quite understand? What do you mean by `ViewModifier`? I used `.environment(\.colorScheme` which is a built in modifier to change SwiftUI to dark mode. Unless you recommend something else.